docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,552 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fe59d027-121a-4411-9dc6-0c1756f205e2",
+   "metadata": {},
+   "source": [
+    "# Alert activations\n",
+    "\n",
+    "There is a way to get all activations of alerts. It can be done in three ways:\n",
+    "1. From run_db\n",
+    "2. For specific project\n",
+    "3. For specific alert config"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "58d2b946-73f7-4f38-8236-4855e4ef4d4e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "> 2024-12-11 11:52:08,031 [warning] Failed resolving version info. Ignoring and using defaults\n",
+      "> 2024-12-11 11:52:09,830 [warning] Server or client version is unstable. Assuming compatible: {\"client_version\":\"0.0.0+unstable\",\"server_version\":\"1.8.0-rc8\"}\n"
+     ]
+    }
+   ],
+   "source": [
+    "import mlrun\n",
+    "\n",
+    "run_db = mlrun.get_run_db()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "d49b918c-b0bf-4f17-862b-f8ae6ee9949e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "activations = run_db.list_alert_activations(\n",
+    "    project=None,\n",
+    "    name=None,\n",
+    "    since=None,\n",
+    "    until=None,\n",
+    "    entity=None,\n",
+    "    severity=None,\n",
+    "    entity_kind=None,\n",
+    "    event_kind=None,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c9ddafb-bb14-4ef4-aa95-3f57e2e0bb64",
+   "metadata": {},
+   "source": [
+    "method returns {py:class}`~mlrun.common.schemas.alert.AlertActivations` object which contains field activations which is the list of {py:class}`~mlrun.common.schemas.alert.AlertActivation` with following fields:\n",
```

Comment:
Does not make sense. Do you men:
"This method returns an {py:class}`~mlrun.common.schemas.alert.AlertActivations` object that contains the field activations, including the following fields:\n",

Reply:
rephrased